src/inspector_io.cc

```diff
@@ -175,15 +198,18 @@ bool InspectorIo::Start() {
     return false;
   }
   state_ = State::kAccepting;
-  if (options_.wait_for_connect()) {
+  if (wait_for_connect_) {
     DispatchMessages();
   }
   return true;
 }
 
 void InspectorIo::Stop() {
+  Write(TransportAction::kKill, 0, StringView());
```

Comment:
I added checks for state. I don't think the io class should be reusable, so once it was stopped it will assert if tried to restart. Will try to implement RAII in the next versions.